packages/design-system/.storybook/webpack.js

```diff
@@ -22,13 +20,56 @@ const replaceSvgLoader = (config) => {
   return { ...config, module: { ...config.module, rules } }
 }
 
-const getConfig = override(
-  replaceSvgLoader,
-  addWebpackAlias({
-    'react-native': 'react-native-web',
-    'react-native-linear-gradient': 'react-native-web-linear-gradient',
-  }),
-)
+const setReactNativeAliases = (config) => ({
+  ...config,
+  resolve: {
+    ...config.resolve,
+    alias: {
+      ...config.resolve.alias,
+      'react-native$': 'react-native-web',
+      'react-native-linear-gradient$': 'react-native-web-linear-gradient',
+    },
+  },
+})
+
+const includeNodeModule = (noduleName) => (config) => ({
+  ...config,
+  module: {
+    ...config.module,
+    rules: [
+      {
+        test: /\.js$/,
+        include: new RegExp(`node_modules\/${noduleName}`),
+        use: [
+          {
+            loader: 'babel-loader',
+            options: {
+              presets: [
+                ['@babel/preset-react', { modules: 'commonjs' }],
+                ['@babel/preset-env', { modules: 'commonjs' }],
+              ],
+            },
+          },
+        ],
+      },
+      ...config.module.rules,
+    ],
+  },
+})
+
+const pipe =
+  (...functions) =>
+  (target) =>
+    functions.reduce((acc, fn) => fn(acc), target)
+
+const getConfig = (config) => {
+  console.log(config.module.rules[3].use[0].options.presets)
+  return pipe(
+    includeNodeModule('react-native-toast-message'),
```

Comment:
`react-native-toast-message` is not transpiled by default. We have to force Webpack to do it.